theme/src/default/_index.scss

```diff
@@ -19,3 +19,5 @@
   --clr-dark: #{vars.$owl-ref-palette-grey-800};
   --clr-dark-highlight: #{vars.$owl-ref-palette-grey-600};
 }
+
```

Comment:
The 'lint:fix` probably added these extra blank lines at the bottom, we don't need it in your PR so you can roll this back.

Same thing for the `web/src/index.html` file below.